pkgs/development/python-modules/redbaron/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchPypi, buildPythonPackage, baron, pytest }:
+
+buildPythonPackage rec {
+  pname = "redbaron";
+  version = "0.9.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "Ry0HOcprIkC7IniuQoYEp1RyycEuhsYyHowBYTnAEy8=";
```

Comment:
There's still a preference to having base32 hashes within nixpkgs to ease the upgrade process for users with holder versions of the nix cli.

You can use `nix-hash --to-base32 <hash>` to convert the hash into the related base32 hash.

For more details refer to: https://github.com/NixOS/nixpkgs/pull/89308#issuecomment-636927054 #89275